### DISCUSSION TITLE
Dozens of identically named files seems like a good idea to you? 😭

### DISCUSSION THREAD

**Discussion by landsman** [vercel/next.js#60382] [App Router]
I hate to see tons of files with the same name. 

It's annoying when you have to find something. It is not productive at all.

Please introduce a way how to define routing in code instead of hacking it via file names. It is really painful for people who are trying to do something more than a "Hello World" blog.

**Comment by icyJoseph** [comment]
Set your VSCode label format property to `short`.

In most other text-editors, this is not necessarily an issue.

**Reply by y-nk** [reply]
solution to a technical issue is "change/tweak editor". wow.
